build.sh

```diff
@@ -141,8 +141,12 @@ build_managed_corefx()
 
 build_native_corefx()
 {
+    # Format the Native Code to make sure it is format-compliant
+    echo "Formatting native source code..."
+    "$__nativeroot/format-code.sh"
```

Comment:
If you run format-code.sh and clang-format doesn't find any issue with a file, the file will not be changed and it does not show up in git status or change the compilation